src/ert/storage/migration/block_fs.py

```diff
@@ -257,7 +257,7 @@ def migrate(path: Path) -> None:
     )
 
 
-def _migrate_case_ignoring_exceptions(storage: Storage, casedir: Path) -> bool:
+def _migrate_case_ignoring_exceptions(storage: "Storage", casedir: Path) -> bool:
```

Comment:
I suggest using `from __future__ import annotations` instead.

Reply:
Yes, the circular dependency issue I had originated elsewhere